src/nbsap/templates/eu_strategy/eu_indicators.html

```diff
@@ -4,69 +4,27 @@
 
 
 {% block title %}
-  NBSAP - {% trans "EU Indicators" %}
+	NBSAP - {% trans "EU Indicators" %}
 {% endblock %}
 
 
 {% block navmenu %}
-  {% include '_bits/nav_menu.html' %}
+	{% include '_bits/nav_menu.html' %}
 {% endblock %}
 
-{% block content %}
-  <h1 class="page-title">{% trans "EU Indicators" %}</h1>
-
-  {% if indicators %}
-    {% for indicator in indicators %}
-
-        <h2 id="indicator-{{ indicator.id }}">
-          {{ indicator.get_indicator_type_display }} {{ indicator.code }} {{ indicator.title }}
-        </h2>
-
-        {% if indicator.url %}
-          <p>URL: <a href="{{ indicator.url }}">{{ indicator.title }}</a></p>
-        {% endif %}
 
-        <div class="well">
-          <div
-              class="well-title">{% trans "Mapping with Aichi Strategy" %}</div>
-          <p><strong>Most relevant:</strong></p>
-          {% if indicator.eu_indicator_aichi_strategy.exists %}
-            <ul>
-              {% for strategy in indicator.eu_indicator_aichi_strategy.all %}
-                {% for aichi_target in strategy.aichi_targets.all %}
-                  {% assign aichi_target.get_parent_goal as goal %}
-                  <li>
-                    <a href="{% url 'goals' code=goal.code %}#{{ aichi_target.code }}">
-                      {% trans "AICHI Target" %} {{ aichi_target.code }}</a>
-                  </li>
-                {% endfor %}
-              {% endfor %}
-            </ul>
-          {% else %}
-            <p>{% trans "No corresponding AICHI target" %}.</p>
-          {% endif %}
-
-          <p><strong>Other:</strong></p>
-          {% if indicator.eu_indicator_other_aichi_strategy.exists %}
-            <ul>
-              {% for strategy in indicator.eu_indicator_other_aichi_strategy.all %}
-                {% for aichi_target in strategy.aichi_targets.all %}
-                  {% assign aichi_target.get_parent_goal as goal %}
-                  <li>
-                    <a href="{% url 'goals' code=goal.code %}#{{ aichi_target.code }}">
-                      {% trans "AICHI Target" %} {{ aichi_target.code }}</a>
-                  </li>
-                {% endfor %}
-              {% endfor %}
-            </ul>
-          {% else %}
-            <p>{% trans "No corresponding AICHI target" %}.</p>
-          {% endif %}
-        </div>
-
-    {% endfor %}
-
-  {% else %}
-    <h1>{% trans 'No EU Indicators' %}</h1>
-  {% endif %}
+{% block content %}
+	<h1 class="page-title">{% trans "EU Indicators" %}</h1>
+
+	{% if indicators %}
+		<div class="indicator-pages">
+			{% for indicator in indicators|sort_by_type_and_code %}
+				<li id="indicator-{{ indicator.id }}">
+					<a href="{% url 'eu_indicator_details' indicator.pk %}">{{ indicator.get_indicator_type_display }} {{ indicator.code }} {{ indicator.title }}</a>
```

Comment:
This is not 2-space indentation. Comment applies to the whole file. This file was actually indented ok, but you changed it.